src/routers/task.js

```diff
@@ -92,4 +94,47 @@ router.delete('/tasks/:id', auth, async(req, res) => {
     }
 })
 
+const upload = multer({
+    limits: {
+        fileSize: 1000000
+    },
+    fileFilter(req, file, callback) {
+        if (!file.originalname.match(/\.(jpg|jpeg|png)$/)) {
+            return callback(new Error('Please upload an image document.'))
+        }
+        callback(undefined, true)
```

Comment:
null, true

Reply:
Could you please clarify what do you mean?

Reply:
```
undefined means that the variable has not been declared, or has not been given a value.
```
you give a value

Reply:
Fixed.

Reply:
same here
https://github.com/sychova/task-manager-api/pull/3/files#diff-9a5883a8755422ad10e76639bef466a705306c197913dcf75e87a549faa6cca6R103

Reply:
Fixed.